### ISSUE TITLE
HttpConnection defaults to TransportType.WebSockets but should default to TransportType.All

### ISSUE THREAD

**Issue by moozzyk** [aspnet/SignalR#574]
Defaulting to TransportType.WebSockets does not work on Windows 7. Defaulting to TransportType.All will make the client pick ServerSentEvents where webSockets are not supported (i.e. Windows 7/ Windows 2008R2) and WebSockets where supported.

I believe we have the same issue in the TS client - browsers on Win7 don't support WebSockets which is the default.

See #573

**Comment by mikaelm12** [comment]
Addressed in : https://github.com/aspnet/SignalR/commit/b2bc4e620739568a42be5c1f57ae3672f6885212

**Comment by jonathanperis** [comment]
I'm having an error right now related to this (developing in Chrome, Windows 7) and even using the TransportType.All setting, my connection returns me the error "Uncaught (in promise) WebSocket is not in the OPEN state."

It was necessary to force the configuration to "TransportType.ServerSentEvents" to work.

Making the connection without specifying a "TransportType" also causes me the error, making me believe that SignalR is not trying to use the ServerSentEvents on the WebSockets fault

I'm using version 1.0.0-alpha2-final of the Nuget package Microsoft.AspNetCore.SignalR and the JS file signalr-client-1.0.0-alpha2-final.min.js

So, in short,

TransportType.All => Is causing me a WebSockets error;
TransportType.WebSockets => Is causing me a WebSockets error;
TransportType.ServerSentEvents => It worked as expected;
